### ISSUE TITLE
Stuck on "Opening Server..."

### ISSUE THREAD

**Issue by herzkoenig** [microsoft/vscode-livepreview#561]
Up until yesterday, everything was working normally. I could execute "Live Preview: Start Server" and it would display the embedded preview. I was using 100% default config.

Today, when I do the same, absolutely nothing visible happens. No preview plane pops up on the right, and also when I change the settings to use external browser previewing nothing happens. Also no output in "Embedded Live Preview".

I tried: restarting VSCode, reinstalling VSCode, reinstalling the extension, using a new profile, wiping all profiles + installed extensions + reinstalling VSCode. Also I tried multiple settings for the extension (various browser, external/internal preview, change port, etc.pp.).
I also tried installing pre-release version of the extension and older versions of the extension, the problem persists.

As I write this, I have also tried the "Start Server Logging" option from the command palette, as well as all other options, and it produced the following output:
![image](https://github.com/microsoft/vscode-livepreview/assets/59410051/a0c25536-cb2d-4b75-9736-faac145ccdd7)

I'm relatively new to VSCode and I suppose there are some additional hidden logs that could help, but I didn't find them.
When you tell me where to get them from, I will do so.

I'm on Windows 11, fully updated using an HP 845 G9 Laptop (AMD Ryzen 5 PRO 6650U).

**Comment by andreamah** [comment]
Are you on a remote session? (ie: remote SSH, codespaces, etc.)

**Comment by herzkoenig** [comment]
> Are you on a remote session? (ie: remote SSH, codespaces, etc.)

No, completely standard installation on fully updated Win 11 Pro (Build 22621.2715).

**Comment by andreamah** [comment]
If you run the `Developer: Toggle Developer Tools` command in the command palette, do you see any errors in the console?

**Comment by herzkoenig** [comment]
I'm not sure why, but it's working now. I didn't intentionally change any settings...

**Comment by jtmcdole** [comment]
I'm running into this problem today (no logging, stuck on `Opening Server...`, no internal preview or external.

Windows 11; VsCode 1.84.2; Live Preview 0.4.11

Runtime status from plugin:
```
Activation Event:onCommand:livePreview.runServerLoggingTask
Activation Time:11ms
```

Developer Tools output after starting server:
```
[Extension Host] Server is running on port 3000
webviewElement.ts:512 An iframe which has both allow-scripts and allow-same-origin for its sandbox attribute can escape its sandboxing.
```

**Comment by andreamah** [comment]
> I'm running into this problem today (no logging, stuck on `Opening Server...`, no internal preview or external.
> 
> Windows 11; VsCode 1.84.2; Live Preview 0.4.11
> 
> Runtime status from plugin:
> 
> ```
> Activation Event:onCommand:livePreview.runServerLoggingTask
> Activation Time:11ms
> ```
> 
> Developer Tools output after starting server:
> 
> ```
> [Extension Host] Server is running on port 3000
> webviewElement.ts:512 An iframe which has both allow-scripts and allow-same-origin for its sandbox attribute can escape its sandboxing.
> ```

Were you running the command `Live Preview: Start Server Logging`? This command doesn't actually open a preview. It only starts the server. Try the command `Live Preview: Show Preview (Internal Browser)` while you're on an HTML file to open a preview (while also running a server).

**Comment by jtmcdole** [comment]
> Were you running the command `Live Preview: Start Server Logging`? This command doesn't actually open a preview. It only starts the server. Try the command `Live Preview: Show Preview (Internal Browser)` while you're on an HTML file to open a preview (while also running a server).

I was originally runting the "Show Preview (Internal Browser)" and it doesn't open any windows in VS Code, it just fails silently. Same for the external option.

Works great on my windows 10 laptop; does not work on my windows 11 desktop.  If there's some flags you want me to run to help out; let me know.

**Comment by andreamah** [comment]
Hmm. If you disable other extensions, does this still happen?

**Comment by jtmcdole** [comment]
I'll give it a try tomorrow and get back to you

**Comment by jtmcdole** [comment]
I disabled all extensions except for the Live Preview:

1. All calls to action for "Show Preview" does not do anything.
2. I can open an external browser to 127.0.0.1:3000 and view the server though.

**Comment by andreamah** [comment]
What do you see in the console when you run `Developer: Toggle Developer Tools` (when you try previewing)?

**Comment by jtmcdole** [comment]
> [Extension Host] Server is running on port 3000
